www/docs/en/dev/config_ref/index.md

```diff
@@ -167,6 +167,17 @@ all of them:
 
         <preference name="BackgroundColor" value="0xff0000ff"/>
 
+  Applies to Windows.
+  
+- CSS Color names are also supported:
+
+```
+<platform name="windows">
+    <preference name="BackgroundColor" value="turquoise"/>
+</platform>
+```
+  __NOTE__: `transparent` value will cause black/white background color in case of Dark/Light theme accordingly.
+
   Applies to Android and BlackBerry. Overrides CSS otherwise available
```

Comment:
Yes, thanks, I've just confused this notation with the plugin docs one where we have a supported platforms list in the start and Quirks in the end.